### PR TITLE
It Adds capability to generate ng1 component.

### PR DESCRIPTION
**Pull request by bernardbr**
Hello @ericmdantas!
I add a capability to generate ng1 component.
But, when I run ```npm t``` an wild error appears :stuck_out_tongue_closed_eyes: 

```console
1) subgenerator -> component ng1 testsSeparated is true "before all" hook:
     Uncaught ReferenceError: appName is not defined
      at eval (lodash.templateSources[1542]:9:11)
      at underscore (node_modules\yeoman-generator\lib\util\engines.js:23:45)
      at ComponentGenerator.actions.engine (node_modules\yeoman-generator\lib\actions\actions.js:224:32)
      at ComponentGenerator.template (node_modules\yeoman-generator\lib\actions\actions.js:198:19)
      at Angular1.copyComponent (C:/Projetos/bernardbr/generator-ng-fullstack/_ng/client/angular.js:73:20)
      at ComponentSubGenerator.writing (C:/Projetos/bernardbr/generator-ng-fullstack/_ng/client/sub_generators_component.js:34:58
)
      at ComponentGenerator.writing (C:/Projetos/bernardbr/generator-ng-fullstack/component/index.js:18:20)
      at node_modules\yeoman-generator\lib\base.js:421:16

  2) subgenerator -> component ng1 testsSeparated is false "before all" hook:
     Uncaught ReferenceError: appName is not defined
      at eval (lodash.templateSources[1543]:9:11)
      at underscore (node_modules\yeoman-generator\lib\util\engines.js:23:45)
      at ComponentGenerator.actions.engine (node_modules\yeoman-generator\lib\actions\actions.js:224:32)
      at ComponentGenerator.template (node_modules\yeoman-generator\lib\actions\actions.js:198:19)
      at Angular1.copyComponent (C:/Projetos/bernardbr/generator-ng-fullstack/_ng/client/angular.js:73:20)
      at ComponentSubGenerator.writing (C:/Projetos/bernardbr/generator-ng-fullstack/_ng/client/sub_generators_component.js:34:58
)
      at ComponentGenerator.writing (C:/Projetos/bernardbr/generator-ng-fullstack/component/index.js:18:20)
      at node_modules\yeoman-generator\lib\base.js:421:16
```

Could you help me? Because I don't know how to resolve this.

Thanks!

### CODE REVIEW

File: component/templates/ng1/component.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+  const TEMPLATE = `
+     <div>
+       <h1>Hello, {{$ctrl.user.name}}</h1>
+     </div>!</p>
+  `;
+
+  ng.module('<%= appName %>', [])
+    .component('<%= name %>', {
+      template: TEMPLATE,
```
Comment (ericmdantas) [inline component/templates/ng1/component.js]: Use: `templateUrl: '<%= feature %>/templates/<%= name %>.html',`

File: component/templates/ng1/component.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+; (function (ng) {
+  'use strict';
+
+  const TEMPLATE = `
```
Comment (ericmdantas) [inline component/templates/ng1/component.js]: not needed, since it'll use templateURL

File: component/templates/ng1/component.html [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<div>
+  <h1>Hello, {{name}}!</h1>
```
Comment (ericmdantas) [inline component/templates/ng1/component.html]: `{{$ctrl.user.name}}`

File: test/acceptance/test-component.js [hunk 4]
```diff
@@ -1,15 +1,73 @@
...
+
+    describe('testsSeparated is false', () => {
+      before(function (done) {
+        helpers
+          .run(path.join(__dirname, '../../component'))
+          .inTmpDir(function (dir) {
+            createYoRc({
+              'generator-ng-fullstack': {
+                'client': 'ng1',
+                'testsSeparated': false
```
Comment (ericmdantas) [inline test/acceptance/test-component.js]: add:

`'appName': 'something'`

File: test/acceptance/test-component.js [hunk 5]
```diff
@@ -1,15 +1,73 @@
...
+  describe('ng1', () => {
+    describe('testsSeparated is true', () => {
+      before(function (done) {
+        helpers
+          .run(path.join(__dirname, '../../component'))
+          .inTmpDir(function (dir) {
+            createYoRc({
+              'generator-ng-fullstack': {
+                'client': 'ng1',
+                'testsSeparated': true
```
Comment (ericmdantas) [inline test/acceptance/test-component.js]: add:

`'appName': 'something'`

**Comment by ericmdantas** [comment]
Thanks!

I've added some comments so you can get the tests running.

**Comment by bernardbr** [comment]
It's done!

**Comment by ericmdantas** [comment]
Thank you!

**Pull request merged**